src/components/DatasetOverviewTab/data/definitions.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+      "pdc": "Modified",
+      "definition": "Most recent date on which the dataset was changed, updated or modified"
+    },
+    {
+      "pdc": "Released",
+      "definition": "xxx"
+    },
+    {
+      "pdc": "Issued",
+      "definition": "Dare of formal issuance\" Project Open Data uses lable \"Release"
+    },
+    {
+      "pdc": "Publisher",
+      "definition": "The publishing entity and optionally their parent organization(s)"
+    },
+    {
+      "pdc": "Identifier",
+      "definition": "A unique identifier for the dataset or API as maintained within an Agency catalog or database"
+    },
+    {
+      "pdc": "Contact",
+      "definition": "Contact name for the asset"
+    },
+    {
+      "pdc": "Contact Email",
+      "definition": "Contact name for the asset"
+    },
+    {
+      "pdc": "Public Access Level",
+      "definition": "The degree to which this dataset could be made pubicly-available. \nChoices:\npublic (data asset is or could be made publicly available to all without restrictions), \nrestricted public (Data aset is available under certain use restrictions) or \nnon-public (Data asset is not available to members of the public"
```

Comment:
The formatting for this one does not match the formatting in the ticket.

Reply:
addressed

https://github.com/GetDKAN/cmsds-open-data-components/pull/239/commits/1c044983ba819884e0d96959948fa37213fb206b

![Screenshot 2024-08-15 at 7 00 59 PM](https://github.com/user-attachments/assets/6ef6d4ac-b4f6-46f9-b0a5-8d0b8925bbe8)